packages/squiggle-lang/__tests__/TestHelpers.res

```diff
@@ -11,17 +11,33 @@ let {toFloat, toDist, toString, toError, fmap} = module(DistributionOperation.Ou
 let fnImage = (theFn, inps) => Js.Array.map(theFn, inps)
 
 let env: DistributionOperation.env = {
-  sampleCount: 100,
-  xyPointLength: 100,
+  sampleCount: 10000,
+  xyPointLength: 1000,
 }
 
 let run = DistributionOperation.run(~env)
 let outputMap = fmap(~env)
 let unreachableInTestFileMessage = "Should be impossible to reach (This error is in test file)"
 let toExtFloat: option<float> => float = E.O.toExt(unreachableInTestFileMessage)
-let toExtDist: option<GenericDist_Types.genericDist> => GenericDist_Types.genericDist = E.O.toExt(
+let toExtDist: option<DistributionTypes.genericDist> => DistributionTypes.genericDist = E.O.toExt(
   unreachableInTestFileMessage,
 )
 // let toExt: option<'a> => 'a = E.O.toExt(unreachableInTestFileMessage)
 let unpackFloat = x => x->toFloat->toExtFloat
 let unpackDist = y => y->toDist->toExtDist
+
+let mkNormal = (mean, stdev) => DistributionTypes.Symbolic(#Normal({mean: mean, stdev: stdev}))
+let mkBeta = (alpha, beta) => DistributionTypes.Symbolic(#Beta({alpha: alpha, beta: beta}))
+let mkExponential = rate => DistributionTypes.Symbolic(#Exponential({rate: rate}))
+let mkUniform = (low, high) => DistributionTypes.Symbolic(#Uniform({low: low, high: high}))
+let mkCauchy = (local, scale) => DistributionTypes.Symbolic(#Cauchy({local: local, scale: scale}))
+let mkLognormal = (mu, sigma) => DistributionTypes.Symbolic(#Lognormal({mu: mu, sigma: sigma}))
```

Comment:
Just flagging that we probably want to use/require use of ``make``, similar to what we discussed earlier. So,  use lines 36-43, not 29-34.

Reply:
(I just realized this was just copied from somewhere else)

Reply:
Definitely!

Reply:
However, in tests I don't think it's essential because test cases are _trusted input_.

What I'd like in worlds where we had a proper property testing library would be for each symbolic dist, write validators in tests that sort out the procgen'd test cases into `Error` paths and `Ok` paths, and validate that assumptions hold. 

But when you say "require", do you mean refactoring preexisting tests? I'd rather leave tests in that aren't up to date then invest in refactoring or deleting them.

Reply:
I think I'm paranoid about allowing exceptions for cases like this; it makes the situation more complicated (in these cases, you can do these things, else, you can do these others). Also, the cost of using ``make`` seems really low to me, anyway. We could call ``E.F.toExn`` after if we trust it. 

> But when you say "require", do you mean refactoring preexisting tests? I'd rather leave tests in that aren't up to date then invest in refactoring or deleting them.

Yea; I'd encourage leaving this for the future. But I think it would be a good refactor at some point.